docs/transpile/defaults-and-configuration-options.mdx

```diff
@@ -65,6 +65,8 @@ All parameters except `circuits` are optional.  For full details, see the [Trans
 
 These arguments describe the system that the input circuit will be run on.  For example, they may be used to ensure that the returned circuit can be run on the specified backend, where the transpiler maps the circuit's virtual qubits to the physical qubits which may have limited connectivity as specified in the `coupling_map`, `basis_gates` that specify which single- and two-qubit gates are calibrated for the backend, and possibly their error rates.
 
+Many of these parameters are described in detail in [Commonly used parameters](common-parameters).
```

Comment:
```suggestion
Many of these parameters are described in detail in [Other commonly used parameters](common-parameters).
```

Reply:
There's a mismatch between the page title and its toc entry. What should it be?

Reply:
I think they should both be "Commonly..."  @javabster ?

Reply:
I deliberately made them different. My logic was that when a user is looking at the sidebar it makes sense to use the word "other" in the context of the other page titles. But if a user just goes directly to the page (e.g. from google search) and isn't paying attention to the other sidebar headings then just reading the page title with the word "other" in it might be coinfusing

Reply:
I don't have a strong feeling though, if you think its better to keep it the same thats fine by me

Reply:
I made the names the same.